modules/custom/activity_creator/src/Service/ActivityCreatorBatchActivityDeletion.php

```diff
@@ -47,7 +48,7 @@ public static function updateProcess(array $items, array &$context): void {
     $activity_storage = \Drupal::entityTypeManager()->getStorage('activity');
 
     // Elements per operation.
-    $limit = 50;
```

Comment:
@nechai I think it's better to keep 50 as default value here.
We also have 50 in other [places in Distro](https://github.com/search?q=repo%3Agoalgorilla%2Fopen_social+%27entity_update_batch_size%27&type=code).
Anyway with this change we will be able to override it per platform.

Reply:
@ribel I would suggest to keep this number as we know that handling "50" items causes the problem (keep in mind, that we aren't deleting the `activity` entities only but `message` entities as well). As you said, it could be changed per platform.